### PR TITLE
chore: GitHub actions

### PR DESCRIPTION
**Pull request by Yserz**
.

### CODE REVIEW

File: .github/workflows/test_build_deploy.yml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+          echo "::set-env name=PR_LAST_COMMIT_MESSAGE::$(git log --format=%B -n 1 ${{github.event.after}})"
+          echo "::set-env name=COMMIT_URL::${{github.event.head_commit.url}}"
+          echo "::set-env name=COMMITTER::${{github.event.head_commit.committer.name}}"
+          echo -e "BRANCH_NAME = ${BRANCH_NAME}"
+          echo -e "TAG = ${TAG}"
+          echo -e "TITLE = ${TITLE}"
+          echo -e "PR_LAST_COMMIT_MESSAGE = ${PR_LAST_COMMIT_MESSAGE}"
+          echo -e "COMMIT_URL = ${COMMIT_URL}"
+          echo -e "COMMITTER = ${COMMITTER}"
+          echo -e "GITHUB_SHA = ${GITHUB_SHA}"
```
Comment (ffflorian) [inline .github/workflows/test_build_deploy.yml:35]: Not used?

File: .github/workflows/test_build_deploy.yml [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+          echo -e "GITHUB_REF = ${GITHUB_REF}"
```
Comment (ffflorian) [inline .github/workflows/test_build_deploy.yml:36]: Not used?

**Review by ffflorian — CHANGES_REQUESTED** [review]

**Review by lipis — APPROVED** [review]

File: .github/workflows/test_build_deploy.yml (hunk 1 above)
Comment (Yserz) [reply .github/workflows/test_build_deploy.yml:35]: This is for debugging

File: .github/workflows/test_build_deploy.yml (hunk 2 above)
Comment (Yserz) [reply .github/workflows/test_build_deploy.yml:36]: This is for debugging

**Review by ffflorian — APPROVED** [review]

**Pull request merged**